internal/lsp/server.go

```diff
@@ -781,7 +782,7 @@ func (l *LanguageServer) handleTextDocumentDefinition(
 	definition, err := orc.FindDefinition(query)
 	if err != nil {
 		// fail silently — the user could have clicked anywhere
-		return struct{}{}, nil //nolint:nilerr
+		return nil, nil //nolint:nilerr
```

Comment:
Do we need the nolint here too? as well as at the top.

Reply:
The one at the top covers the whole file.

Reply:
But we don't need `//nolint:nilerr` here right?

Reply:
We do, as `nilnil` is the one we ignore on the file.

```
internal/lsp/server.go:787:3: error is not nil (line 784) but it returns nil (nilerr)
		return nil, nil
```

That one seems to check for return of nil while `err != nil`